automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/NotebookInteractionSpec.scala

```diff
@@ -303,6 +303,27 @@ class NotebookInteractionSpec extends FreeSpec with LeonardoTestUtils with Befor
       }
     }
 
+    // See https://github.com/DataBiosphere/leonardo/issues/398
+    "should be able to install mlr" in withWebDriver { implicit driver =>
+      Orchestration.billing.addUserToBillingProject(billingProject.value, ronEmail, Orchestration.billing.BillingProjectRole.User)(hermioneAuthToken)
+
+      withNewNotebook(ronCluster, RKernel) { notebookPage =>
+        // mlr: machine learning in R
+        // https://github.com/mlr-org/mlr
+
+        // it may take a little while to install
+        val installTimeout = 2.minutes
+
+        val installOutput = notebookPage.executeCell("""devtools::install_github("mlr-org/mlr")""", installTimeout)
+        installOutput should be 'defined
+        installOutput.get should include ("Installing mlr")
```

Comment:
Does `Installing mrl` get displayed as part of the output even if the installation fails? If so, the check above may give false confidence and might better check something unique to successful installation.

Reply:
It does -- in fact the output is identical in both success and failure cases except that the failure includes an extra "Installation failed: Command failed (1)". 

So the check for "Installing mlr" may be a bit superfluous but the subsequent checks should still verify that it installed correctly.

Reply:
I agree that the subsequent checks should suffice. I would remove the line 319 since it doesn't seem to add any value.

Reply:
Well, on the other hand it still validates that the install was kicked off (even if it doesn't indicate success/failure). If `devtools::install_github()` broke for whatever reason it would cause the check to fail. I like having at least one positive check of the install output.

Reply:
Makes sense to keep then 👍